src/components/EventCard/index.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { Link } from "gatsby";
+import { IoIosArrowRoundForward } from "react-icons/io";
+import { BiLinkExternal } from "react-icons/bi";
+import Image from "../image";
+
+import { CardWrapper } from "./Card.style";
+
+const Card = ({ frontmatter, fields }) => {
+    return (
+        <CardWrapper fixed={!!frontmatter.abstract}>
+            <div className="post-block">
+                <div className="post-thumb-block">
+                    <a className="anchor" href={frontmatter.eurl}>
```

Comment:
I think it would be nice when the user clicks on the *title/card* It should open in a separate tab other than opening in the same tab. @Jashpatel1 @Nikhil-Ladha Your thoughts?

Reply:
Not necessary I guess.

Reply:
I think that would be better, @vinayaksh42 can tou send out a seperate PR for that?

Reply:
@Nikhil-Ladha sure